src/firefly/js/templates/lightcurve/LcUtil.jsx

```diff
@@ -3,14 +3,18 @@ import PropTypes from 'prop-types';
 import {get, has, set, isEmpty} from 'lodash';
```

Comment:
Same here.  It looks like you took other people changes as your own.

Reply:
No idea what happened.  In my local branch, it is fine.  It showed the last modification is July 19.  I also compared with dev.